website/blog/history.md

```diff
@@ -0,0 +1,466 @@
+# A History of Service Weaver's Core API
+
+<div class="blog-author">Michael Whittaker</div>
+<div class="blog-date">TODO</div>
+
+In this blog post, we explore the history of Service Weaver's core
+[components][components] API. The API has steadily evolved since the inception
+of Service Weaver over two years ago.  We hope that by studying the API's
+history, you gain a better understanding of
+
+- why the API is the way it is,
+- how it differs from alternative APIs, and
+- what the API might look like in the future.
+
+We conclude with some lessons learned on API design.
+
+## September 30, 2021: In the Beginning...
```

Comment:
Could benefit from an brief listing of all changes at the top (one line per change). Skimmers can just go with that.

Maybe table of conntents will show up automatically and provide that listing?

Reply:
Good idea! Done.